src/pages/home/_hero.tsx

```diff
@@ -76,7 +78,7 @@ const StyledHeader = styled(Header)`
     }
 `
 
-const Hero = ({ is_ppc }: HeroProps) => {
+const Hero = ({ is_ppc, is_logged_in, hide_signup_login }: HeroProps) => {
```

Comment:
In case when we are didn't pass `is_logged_in , hide_signup_login ` to <Hero /> in homepage these should always been false. How we'll handle if user is already logged in?